openage/convert/processor/export/media_exporter.py

```diff
@@ -122,6 +123,12 @@ def _export_blend(export_request, sourcedir, exportdir, blend_mode_count=None):
                 exportdir[export_request.targetdir],
                 f"{export_request.target_filename}{idx}.png"
             )
+        source_format = export_request[export_request.source_filename.rindex("."):]
+        source_size = os.path.getsize(source_file)
+        target_size = os.path.getsize(export_request.targetdir+export_request.target_filename)
+        logging.basicConfig(level=logging.DEBUG)
+        logging.debug(f"Source File:{export_request.source_filename} ({source_format}, {source_size}) -> Target File:{export_request.target_filename} (PNG, {target_size})")
```

Comment:
Looks good to me :)

Instead of hardcoding `PNG` as the target format, you could also use the `suffix` property of `target_file` here. This way, the code becomes more future proof if we decide to add another target format.